src/playground/Widgets/List/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { ProgressBar } from "@/components/Charts/ProgressChart"
+
+interface BaseListItem {
+  name: string
+}
+
+interface DefaultListItem extends BaseListItem {
+  value: number | string
+}
+
+interface ProgressListItem extends BaseListItem {
+  value: number
+}
+
+type ListItem = DefaultListItem | ProgressListItem
+
+type ListProps = {
+  title?: string
+  items: ListItem[]
+}
+
+const ListItem = ({ item }: { item: ListItem }) => (
+  <div className="mb-2 flex w-full items-center gap-4 last:mb-0">
+    <span className="flex-grow truncate text-muted-foreground">
+      {item.name}
+    </span>
+    {typeof item.value === "number" ? (
```

Comment:
If we automatically decide to show a progress bar, we assume that a number passed is in percents. This is not clear from the API and it could be surprising for the user. 

Shouldn't we have a `variant` or `type` field that will tell us how to represent the passed number?